internal/installercache/installercache.go

```diff
@@ -310,8 +325,10 @@ func (i *Installers) evictFile(filePath string) error {
 	i.log.Infof("evicting binary file %s due to storage pressure", filePath)
 	err := os.Remove(filePath)
 	if err != nil {
+		i.metricsAPI.InstallerCacheReleaseEvicted(false)
```

Comment:
Shouldn't we measure this in the outer method `evict()`? https://github.com/openshift/assisted-service/pull/7281/files#diff-cf73de10b668273452bf6aeb594f0385d3f3ea5b1a851c0f577d2d8989e6cccdR320

If we really want to add number of files evicted we should change the metric type to bucketed

Reply:
The intention here is just to show evictions are occurring, maybe we can make do with logs, I'll remove this metric.